src/layouts/Layout.astro

```diff
@@ -92,17 +93,9 @@ const { title, headerProps, isVisible } = Astro.props;
       content="https://opengraph.b-cdn.net/production/documents/be8ca05b-5de1-4ab8-99b6-b064c8ad0b4c.png?token=Jriat_jsVklKiXah09FNuY3b8X2kT-Vv-3eN4Cgdyvg&height=675&width=1200&expires=33241899715"
     />
   </head>
-  <body class="bg-white select-none">
-    <nav
-      class="fixed left-0 top-0 z-50 w-full bg-black bg-opacity-[0.5] p-4 backdrop-blur-sm"
-    >
-      <div class="m-auto flex w-full justify-between">
-        <a href="/">
-          <img src="/footer/logo.png" class="h-16" />
-        </a>
-        <Hamburger client:only />
-      </div>
-    </nav>
+  <body class="bg-white">
+    <Navbar client:load />
```

Comment:
Lets hope `client:load` works once its deployed to github pages. I had to do `client:only` because the pipeline failed. Dont change this, just remember this if build fails